mixins/_icons.scss

```diff
@@ -69,16 +73,19 @@
         content: "\64";
     }
     @if $symbol == 'down' {
-        content: "\76";
+        content: "\56";
     }
     @if $symbol == 'download' {
         content: "\2913";
     }
+    @if $symbol == 'expand' {
+        content: "\21f2";
+    }
     @if $symbol == 'eye' {
         content: "\69";
     }
     @if $symbol == 'football' {
-        content: "\1f3c8";
+        content: "\61";
```

Comment:
This is used in the NFL IA: https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/sports/nfl/games/nfl_score.handlebars#L9

It's used via the class, which isn't affected, so this should be fine.